volontulo/templates/offers/offer_apply.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <label class="col-xs-1 control-label" for="{{ form.email.id_for_label }}">Email:</label>
             <div class="col-xs-4">
-               <input type="text" value="{{ request.POST.email|default:volunteer_user.user.email }}" name="email" class="form-control" id="{{form.email.id_for_label }}"{% if user.is_authenticated %}disabled="disabled"{% endif %} />
```

Comment:
In my opinion more properly would be to write readonly="readonly". It's not required but more aesthetic.